packaging/arch/PKGBUILD

```diff
@@ -0,0 +1,254 @@
+	for _cmp in ${_components[@]}; do
+		make \
+			BASEDIR="$pkgdir" \
+			PREFIX=/usr \
+			LIBDIR=lib \
+			install-app
+
+		# Conforms to the arch packaging standards (https://wiki.archlinux.org/index.php/Arch_Packaging_Standards)
+		mkdir -p "$pkgdir"/etc/
+		mv "$pkgdir"/usr/etc/opensips/ "$pkgdir"/etc/
+		sed -i 's#mpath=".*lib/opensips/modules/"#mpath="/usr/lib/opensips/modules/"#' "$pkgdir"/etc/opensips/opensips.cfg
+
+		# fix bad paths
+		cd "$pkgdir"/usr/share
+		find -type f -exec sed -i "s#"$pkgdir"##" {} \;
+
+		mv "$pkgdir"/usr/sbin "$pkgdir"/usr/bin
+
+		cd "$pkgdir"/etc
+		find -type f -exec sed -i "s#"$pkgdir"##" {} \;
+
+		# python2 is being used
+		cd "$pkgdir"
+		grep -lR '#!.*python' * | while read file; do sed -s 's/\(#!.*python\)/\12/g' -i "$file"; done
+
+		# Systemd files
+		install -Dm0644 "${srcdir}/${pkgbase%%-*}-${pkgver}/packaging/arch/${pkgname}.service" "$pkgdir"/usr/lib/systemd/system/${pkgname}.service
+		install -Dm0644 "${srcdir}/${pkgbase%%-*}-${pkgver}/packaging/arch/${pkgname}.tmpfiles.conf" "$pkgdir"/usr/lib/tmpfiles.d/${pkgname}.conf
+	done
+}
+
+package_opensips-modules() {
+	pkgdesc="OpenSIPS an open source SIP Server - modules (stable version)"
+	depends=('opensips')
+	provides=("opensips-modules=${pkgver}")
+	conflicts=('opensips-modules-git')
+
+	cd "$srcdir"/${pkgbase%%-*}-${pkgver}
+
+	make \
+		BASEDIR="$pkgdir" \
+		PREFIX=/usr \
+		LIBDIR=lib \
+		install-modules
+
+	msg2 "ensure binaries live in /bin and /usr/bin"
+	mv "$pkgdir"/usr/sbin "$pkgdir"/usr/bin
+
+}
+
+package_opensips-documentation() {
+	pkgdesc="OpenSIPS an open source SIP Server - documentation (stable version)"
+	pkgdesc="Latest stable version of OpenSIPS documentation"
+	suggests=('opensips')
+	provides=("opensips-documentation=${pkgver}")
+	conflicts=('opensips-documentation-git')
+
+	cd "$srcdir"/${pkgbase%%-*}-${pkgver}
+
+	msg2 "create documentation targets"
+	make \
+		BASEDIR="$pkgdir" \
+		PREFIX=/usr \
+		LIBDIR=lib \
+		doxygen modules-docbook-html
+		# modules-docbook-pdf \
+		#dbschema-docbook-html dbschema-docbook-pdf \
+		#modules-readme
+
+	msg2 "install documentation targets"
+	make \
+		BASEDIR="$pkgdir" \
+		PREFIX=/usr \
+		LIBDIR=lib \
+		install-doc \
+		install-modules-docbook
+
+	msg2 " ... text files"
+	DOC_DIR="$pkgdir/usr/share/doc/${pkgbase%%-*}"
+	if [ -d "$DOC_DIR/txt" ]; then
+		mkdir -p "$DOC_DIR/txt"
+		chmod 0755 "$DOC_DIR/txt"
+		mv --recursive $DOC_DIR/README.* "$DOC_DIR/txt"
+		chmod --recursive 0644 "$DOC_DIR/txt"
+	fi
+
+	msg2 " ... html documents"
+	DOC_DIR="$pkgdir/usr/share/doc/${pkgbase%%-*}"
+	if [ -d "$DOC_DIR/html" ]; then
+		mkdir -p "$DOC_DIR/html"
+		chmod 0755 "$DOC_DIR/html"
+		mv --recursive $DOC_DIR/*.html "$DOC_DIR/html"
+		chmod --recursive 0644 "$DOC_DIR/html"
+	fi
+
+	msg2 " ... examples"
+	DOC_DIR="$pkgdir/usr/share/doc/${pkgbase%%-*}"
+	if [ -d ./examples ]; then
+		mkdir -p "$DOC_DIR"
+		cp --recursive ./examples "$DOC_DIR"
+		chmod --recursive 0644 "$DOC_DIR"
+	fi
+}
```

Comment:
I see that there already is an official package for Arch Linux [here](https://www.archlinux.org/packages/community/x86_64/opensips/). Is this one different than that one? Or why do we need a different packaging, for the `git` repo?